Makefile

```diff
@@ -69,7 +69,7 @@ bootstrap: \
 build: ## build the app container
 	@$(COMPOSE_BUILD) base;
 	@$(COMPOSE_BUILD) app;
-	@$(COMPOSE_BUILD) e2e;
+	@$(COMPOSE_BUILD) --no-cache e2e;
```

Comment:
😭

Reply:
is it really needed ?

Reply:
This is how I get the same error as circle ci…
But maybe we could use --no-cache only for `make build-e2e` ?

Reply:
yes it should be great. This is because the base image was not the same ?

Reply:
I think using --no-cache is a way to be closest as possible to circle-ci.

Reply:
I don't think base image is involved here, only dependencies version.

Reply:
Let me try another modification for running the same image in circle and locally.
I'll request another review if everything seems ok locally.

Reply:
I feel like we should keep a e2e image close to the OS that is running docker.
Especially when we run headed tests.
I'll remove the commit that uses the same image as circle-ci, and keep that in mind for further problems.